index.js

```diff
@@ -308,9 +312,16 @@ async function run() {
         throw new Error(`Service is ${serviceResponse.status}`);
       }
 
+      // Get Desired count of the service before deployment
+      const desiredCountResponse = serviceResponse.desiredCount;
+      if (desiredCountResponse === 0) {
+        core.warning(`Service have desired-count ${desiredCountResponse}, set property 'desired-count' to \
+        start you service if required`);
+      }
+
```

Comment:
No need for this since ecs update service is already stateful. If you leave any param out it'll keep the current value.